### ISSUE TITLE
msiDataObjRsync between zones does not fire remote rules

### ISSUE THREAD

**Issue by cookie33** [irods/irods#2993]
Hi,

If we do on zone alice as the rods user robertv:
```
iput bla source.txt
irule -F rulemsiDataObjRsync.r
Source object /alice/home/robertv/source.txt synchronized with destination object /alice/home/robertv/dest_resc.txt
status = 99999992
```
The following rule is fired in core.re:
```
acPostProcForCopy {
        ON($filePath like "/opt/testVault/*") {
                        writeLine("serverLog","Execute command to replicate $objPath to eudat using eudat, because of copy in testResource");
        }
}
```
The executing rule is:
```
rulemsiDataObjRsync {
#Input parameters are:
#  Data object path
#  Optional flag for mode
#    IRODS_TO_IRODS
#    IRODS_TO_COLLECTION
#  Optional storage resource
#  Optional target collection
#Output parameters are:
#  status
#Output from running the example is:
# Source object /tempZone/home/antoine/source.txt synchronized with destination object /tempZone/home/antoine/dest.txt
  msiDataObjRsync(*sourceObj,"IRODS_TO_IRODS",*destResc,*destObj,*status);
  writeLine("stdout","Source object *sourceObj synchronized with destination object *destObj");
  
  writeLine("stdout","status = *status");
  
}
INPUT *sourceObj="/alice/home/robertv/source.txt",*destResc="null",*destObj="/alice/home/robertv/dest_resc.txt"
OUTPUT ruleExecOut
```

If we do on zone bob as the rods user robertv:
```
iput bla source.txt
irule -F rulemsiDataObjRsync.r
Source object /bob/home/robertv/source.txt synchronized with destination object /alice/home/robertv#bob/dest.txt
status = 99999992
```
There is NO rule fired on the zone alice.

The executing rule is:
```
rulemsiDataObjRsync {
#Input parameters are:
#  Data object path
#  Optional flag for mode
#    IRODS_TO_IRODS
#    IRODS_TO_COLLECTION
#  Optional storage resource
#  Optional target collection
#Output parameters are:
#  status
#Output from running the example is:
# Source object /tempZone/home/antoine/source.txt synchronized with destination object /tempZone/home/antoine/dest.txt
  msiDataObjRsync(*sourceObj,"IRODS_TO_IRODS",*destResc,*destObj,*status);
  writeLine("stdout","Source object *sourceObj synchronized with destination object *destObj");
  
  writeLine("stdout","status = *status");
  
}

INPUT *sourceObj="/bob/home/robertv/source.txt",*destResc="null",*destObj="/alice/home/robertv#bob/dest.txt"
OUTPUT ruleExecOut
```

All objects end up in the same resource. Why is it that across zones the rule is not fired?
As a side note: If we use `irsync  i:/bob/home/robertv/source.txt  i:/alice/home/robertv#bob/dest.txt` the rule is fired

**Comment by adetorcy** [comment]
Thanks for reporting this. We will replicate this in our testing environment to confirm it's a bug and include a fix in 4.1.8

**Comment by jasoncoposky** [comment]
i have confirmed that a manual irsync triggers the PEP, but a call to the microservice via irule does not.
